Makefile

```diff
@@ -75,18 +76,14 @@ unittests:
 
 functional: build
 	@echo "Executing functional tests using built charm at ${PROJECTPATH}"
-	@CHARM_LOCATION=${PROJECTPATH} tox -e func -- ${FUNC_ARGS}
+	@TEST_JUJU_CHANNEL=2.9/stable CHARM_LOCATION=${PROJECTPATH} tox -e func -- ${FUNC_ARGS}
 
-functional33-jammy: build
-	@echo "Executing functional tests using built charm at ${PROJECTPATH} with juju 3.3 requirements"
-	@CHARM_LOCATION=${PROJECTPATH} tox -e func33-jammy -- ${FUNC_ARGS}
-
-functional33-focal: build
-	@echo "Executing functional tests using built charm at ${PROJECTPATH} with juju 3.3 requirements"
-	@CHARM_LOCATION=${PROJECTPATH} tox -e func33-focal -- ${FUNC_ARGS}
+functional3: build
+	@echo "Executing functional tests using built charm at ${PROJECTPATH} with juju 3.x requirements"
+	@TEST_JUJU_CHANNEL=${TEST_JUJU_CHANNEL} TEST_JUJU3=1 CHARM_LOCATION=${PROJECTPATH} tox -e func3 -- ${FUNC_ARGS}
```

Comment:
Could we add a comment explaining the usage of `TEST_JUJU3` similar to [Robert's juju 3.4 support PR](https://github.com/canonical/charm-cloudsupport/pull/21/files)?